introduction.md

```diff
@@ -0,0 +1,2 @@
+Hello. This is a test repo, forking a simple repo, and adding a file. Woohoo.
```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.